python/lsst/pipe/drivers/skyCorrection.py

```diff
@@ -56,13 +57,44 @@ def makeCameraImage(camera, exposures, filename=None, binning=8):
     return image
 
 
+def _skyLookup(datasetType, registry, quantumDataId, collections):
+    """Lookup function to identify sky frames
+
+    Parameters
+    ----------
+    datasetType : `str`
+        Dataset to lookup.
+    registry : `lsst.daf.butler.Registry`
+        Butler registry to query.
+    quantumDataId : `lsst.daf.butler.ExpandedDataCoordinate`
```

Comment:
```suggestion
    quantumDataId : `lsst.daf.butler.DataCoordinate`
```

(`ExpandedDataCoordinate` used to exist, but now there's only one public `DataCoordinate` class.)